packages/cf-deploy-config-writer/src/translations/cf-deploy-config-writer.i18n.json

```
@@ -1,9 +1,11 @@
 {
     "debug": {
         "logError": "{{method}} error found {{ error }}",
```

```suggestion
        "logError": "{{method}} error found: {{ error }}",
```
What should do to try to resolve this? View the logs? Try again?